server/src/modules/users/domain/value-objects/auth-code-string.ts

```diff
@@ -1,25 +1,25 @@
 import crypto from 'crypto'
 
 export class AuthCodeString {
-  private value: string;
-  
+  private value: string
+
   public constructor(hashedValue?: string) {
-    if(hashedValue){
+    if (hashedValue) {
       this.value = hashedValue
     } else {
       this.value = this.getRandomCode()
     }
   }
 
   private getRandomCode() {
-    /*motivation for a 256 bit (= 32 byte) crypographic key can be found here
+    /*motivation for a 256 bit (= 32 byte) cryptographic key can be found here
       https://www.geeksforgeeks.org/node-js-crypto-randombytes-method/
     */
     const authCodeBuffer = crypto.randomBytes(32)
     return authCodeBuffer.toString('hex')
   }
```

Comment:
this function can just be an unexported global function in this file, rather than a member method